force-app/main/default/classes/AdjudicationTest.cls

```diff
@@ -4,32 +4,22 @@ public with sharing class AdjudicationTest {
     @isTest
     static void test_action() {
         SpecialAuthorityRequest sar = TestFactory.newSpecialAuthorityRequest();
-        StepAction stepAction = newAction('Approved');
-
+		Step_Action__c record = AdjudicationTest.getStepActionRecord();
+        StepAction action = new StepAction(record);
         Test.startTest();
-        Action action = new Adjudication();
-        action.execute(sar, stepAction);
+         action.run(sar);
         Test.stopTest();
 
-        system.assertEquals(sar.saCase.Status, stepAction.adjudicationStatus);
+        system.assertEquals(sar.saCase.Status, 'Approved');
```

Comment:
Tech-debt, missing message parameters for system assert statements https://pmd.github.io/pmd/pmd_rules_apex_bestpractices.html#apexassertionsshouldincludemessage

Reply:
@cgiSunilM  Updated the assert statement with related message